CentOS-Linux-AppStream.repo

```diff
@@ -0,0 +1,20 @@
+# CentOS-Linux-AppStream.repo
+#
+# The mirrorlist system uses the connecting IP address of the client and the
+# update status of each mirror to pick current mirrors that are geographically
+# close to the client.  You should use this for CentOS updates unless you are
+# manually picking other mirrors.
+#
+# If the mirrorlist does not work for you, you can try the commented out
+# baseurl line instead.
```

Comment:
do we need this comment? because I don't understand its relation to being able to add our dependencies